ecl/hql/hqlgram.y

```diff
@@ -10308,14 +10309,12 @@ dsOption
                         {   $$.setExpr(createExprAttribute(maxCountAtom, $3.getExpr()), $1); }
     | AVE '(' constExpression ')'
                         {   $$.setExpr(createExprAttribute(aveAtom, $3.getExpr()), $1); }
-    | VIRTUAL '(' UNKNOWN_ID ')'
+    | __OPTION__ '(' hintList ')'
```

Comment:
Syntax for overriding on a single file/index changed.  VIRTUAL wasn't really a good name.  The idea is that __OPTION__ may eventually become a low level compiler-hint equivalent.  (The VIRTUAL syntax was introduced in 7.0 so introduces no backward compatibility issues.)

Reply:
added one HPCC-20296